Library/src/Multiplayer/SpaceEntity.cpp

```diff
@@ -145,6 +145,11 @@ const csp::common::String& SpaceEntity::GetName() const
 
 void SpaceEntity::SetName(const csp::common::String& Value)
 {
+	if (!IsModifiable())
+	{
+		return;
```

Comment:
Should we print errors to the log as otherwise this may come cross to the client application a silent (and unexpected) failures? Appreciate that's a lot of code repetition though...